README.md

```diff
@@ -3,25 +3,15 @@
 **rustls-native-certs** allows [rustls](https://github.com/rustls/rustls) to use the
 platform's native certificate store when operating as a TLS client.
 
-This is supported on Windows, macOS and Linux:
-
-- On all platforms, the `SSL_CERT_FILE` environment variable is checked first.
-  If that's set, certificates are loaded from the path specified by that variable,
-  or an error is returned if certificates cannot be loaded from the given path.
-  If it's not set, then the platform-specific certificate source is used.
-- On Windows, certificates are loaded from the system certificate store.
-  The [`schannel`](https://github.com/steffengy/schannel-rs) crate is used to access
-  the Windows certificate store APIs.
-- On macOS, certificates are loaded from the keychain.
-  The user, admin and system trust settings are merged together as documented
-  by Apple.  The [`security-framework`](https://github.com/kornelski/rust-security-framework)
-  crate is used to access the keystore APIs.
-- On Linux and other UNIX-like operating systems, the
-  [`openssl-probe`](https://github.com/alexcrichton/openssl-probe) crate is used to discover
-  the filename of the system CA bundle.
+> [!IMPORTANT]
+> Instead of this crate, we suggest using [rustls-platform-verifier](https://github.com/rustls/rustls-platform-verifier)
+> instead, which provides a more robust solution with a simpler API. This crate is still
```

Comment:
Two "instead"s in this sentence.
```suggestion
> Instead of this crate, we suggest using [rustls-platform-verifier](https://github.com/rustls/rustls-platform-verifier),
> which provides a more robust solution with a simpler API. This crate is still
```